### PR TITLE
Added x-mx-odata properties for some resources

### PR DESCRIPTION
**Pull request by vrdmr**
This work is required for the cmdlet migration work we are doing to move to Swagger based SDK.

----
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/hybridRunbookWorkerGroup.json [hunk 1]
```diff
@@ -255,7 +255,8 @@
         },
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
-        }
+        },
+        "x-ms-odata": "./definitions.json#/definitions/HybridRunbookWorkerGroup"
```
Comment (mcardosos) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/hybridRunbookWorkerGroup.json]: >HybridRunbookWorkerGroup [](start = 55, length = 24)

Also add the $filter parameter

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/jobSchedule.json [hunk 2]
```diff
@@ -258,7 +258,8 @@
         },
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
-        }
+        },
+        "x-ms-odata": "./definitions.json#/definitions/JobSchedule"
```
Comment (mcardosos) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/jobSchedule.json]: >JobSchedule [](start = 55, length = 11)

`$filter` parameter is missing

Comment (vrdmr) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/jobSchedule.json]: Added.

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/hybridRunbookWorkerGroup.json (hunk 1 above)
Comment (vrdmr) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/hybridRunbookWorkerGroup.json]: Added.

**Comment by vrdmr** [comment]
@mcardosos Could you please review this PR. Thanks, Varad.

**Review by mcardosos — APPROVED** [review]
For awareness, there are [linter errors](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/373230314#L738). I know they were not introduced in this PR.

**Pull request merged**